### PR TITLE
Use configured dropletRepository as given

### PR DESCRIPTION
**Pull request by kieron-dev**
## Is there a related GitHub Issue?
#1914

## What is this change about?
Do not append a suffix, as this means we cannot support registries which disallow dynamic repository creation on push (e.g. ECR).

Rename some config values to make this clear to operators:

- Helm value in kpack-image-builder:
  - `dropletRepositoryPrefix` --> `dropletRepository`
- kpack-image-builder config map property
  - `kpackImageTag` --> `dropletRepository`

Finally, update docs.

## Does this PR introduce a breaking change?
Yes. Korifi packagers and users should note the two renames above.

## Acceptance Steps
1. Deploy korifi successfully using the new helm value in the kpack-image-builder chart.
2. Successfully push an app.

### CODE REVIEW

File: helm/kpack-image-builder/templates/cluster-builder.yaml [hunk 1]
```diff
@@ -3,6 +3,8 @@ apiVersion: kpack.io/v1alpha2
 kind: ClusterStore
 metadata:
   name: cf-default-buildpacks
+  annotations:
+    korifi.cloudfoundry.org/helm-release-version: {{ .Release.Revision | quote }}
```
Comment (danail-branekov) [inline helm/kpack-image-builder/templates/cluster-builder.yaml:7]: Is that annotation related?

Comment (kieron-dev) [reply helm/kpack-image-builder/templates/cluster-builder.yaml:7]: Ah no. That was me trying to fix yesterday's kpack problems. I'll remove it.

**Review by danail-branekov — APPROVED** [review]
